sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/subscriptions/SubscriptionsDbConnector.groovy

```diff
@@ -116,4 +120,32 @@ class SubscriptionsDbConnector implements SubscriptionDataSource {
           }
           return personId
     }
+
+    private boolean projectAlreadySubscribed(int subscriberId, String projectCode) {
```

Comment:
you can also return here the id instead of a boolean and -1 if not found

Reply:
I'd rather keep the boolean since it's more intuitive